### PR TITLE
update info about immutable import

### PR DESCRIPTION
**Pull request by techgaun**
Since the fix has landed and is released, I guess we can update that part now.

### CODE REVIEW

File: tutorial/05-redux-immutable-fetch.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ console.log(immutablePerson)
 
 - Run `yarn add immutable`
 
-**Note**: Due to the implementation of ImmutableJS, Flow does not accept importing it with `import Immutable from 'immutable'`, so use this syntax instead: `import * as Immutable from 'immutable'`. Let's cross fingers for a [fix](https://github.com/facebook/immutable-js/issues/863) soon.
+**Note**: If you are using ImmutableJS <= v3.8.1, Flow does not accept importing it with `import Immutable from 'immutable'`, so use this syntax instead: `import * as Immutable from 'immutable'`. The fix has landed in [v4.0.0-rc.1](https://github.com/facebook/immutable-js/releases/tag/v4.0.0-rc.1) and you don't need to use the alternative syntax for the versions >= v4.0.0-rc.1. `yarn add immutable` automatically installs the correct version.
```
Comment (verekia) [inline tutorial/05-redux-immutable-fetch.md]: Let's remove this note completely :)

File: tutorial/05-redux-immutable-fetch.md [hunk 2]
```diff
@@ -75,15 +75,16 @@ This file exposes an *action*, `SAY_HELLO`, and its *action creator*, `sayHello`
...
+import type { fromJS } from 'immutable'
 
 import { SAY_HELLO } from '../action/hello'
 
 const initialState = Immutable.fromJS({
   message: 'Initial reducer message',
 })
 
-const helloReducer = (state: Object = initialState, action: { type: string, payload: any }) => {
+const helloReducer = (state: fromJS = initialState, action: { type: string, payload: any }) => {
```
Comment (verekia) [inline tutorial/05-redux-immutable-fetch.md]: Let's rename this type to make it clearer:

`import type { fromJS as Immut } from 'immutable'`

`state: Immut`

**Comment by techgaun** [comment]
@verekia thanks for the review. I updated the PR as per your feedback and squashed the commits as well. Could you please check again?

**Comment by verekia** [comment]
Thank you!

**Pull request merged**